setup.cfg

```diff
@@ -35,7 +35,9 @@ line_length=88
 # custom configurations
 order_by_type=False
 skip_glob = **/*_pb2.py
-known_local_folder=packages,tests
```

Comment:
this will make sure that imports from `packages` will be in their own import group.

That is:
```
import standard_lib

import third_party

import aea.some.package

import packages.author.type.name

import local_folder
```

Reply:
Great!